### ISSUE TITLE
The "Responsive contexts require Breakpoint" warning is displayed when no responsive context are defined/used

### ISSUE THREAD

**Issue by lolmaus** [at-import/Singularity#203]
Hi!

I'm building a fixed-width page with no media queries.

I decided not to import Breakpoint.

Example grid definition:

``` sass
+add-grid(12)
+add-gutter(0.1)
```

Example usage:

``` sass
.foo
  +grid-span(1,1)
```

This line alone produces four identical warnings: 

> WARNING: Responsive contexts require Breakpoint (https://github.com/Team-Sass/breakpoint). Please ensure that Breakpoint is imported and available for Singularity to use. First item used.

How do i use Singularity without Breakpoint?

**Comment by lolmaus** [comment]
Using `column-span()` is impossible without Breakpoint, it raises the same warning as an error.

**Comment by Snugug** [comment]
What version of Singularity are you using? I can't reproduce this issue with [ruby](http://sassmeister.com/gist/f8ee0fc775f05e3a17f2) or [libsass](http://sassmeister.com/gist/8d27e3b55f2749bace98) with 1.5.1

**Comment by lolmaus** [comment]
That was on 1.4.

I've upgraded to 1.5.1. `column-span()` no longer errors out, but i still see numerous warnings as described in the top post.

I'm on ruby 2.1.5p273 and using Compass, but assets are fetched via Bower and made available to Compass via `add_import_path 'bower_components/singularity/stylesheets'`.

I've commented out all code from my root Sass file and added this instead:

``` sass
@import singularitygs

.foo
  +grid-span(1,1)
```

This results in this warning appearing four times:

> WARNING: Responsive contexts require Breakpoint (https://github.com/Team-Sass/breakpoint). Please ensure that Breakpoint is imported and available for Singularity to use. First item used.
>          on line 43 of /home/lolmaus/Code/hb_admin_panel_ember/bower_components/singularity/stylesheets/singularitygs/helpers/_find.scss, in `grid-span'
>          from line 9 of /home/lolmaus/Code/hb_admin_panel_ember/app/styles/app.sass

The error is raised from the [`find-object` function](https://github.com/at-import/Singularity/blob/c2ef7b6bcf7915ac460ab9021c3a743684ad4fe5/stylesheets/singularitygs/helpers/_find.scss#L43). The comment above the function admits that it will fail without Breakpoint.

But i can see that `grid-span` [unconditionally calls](https://github.com/at-import/Singularity/blob/c2ef7b6bcf7915ac460ab9021c3a743684ad4fe5/stylesheets/singularitygs/_api.scss#L13) `find-grid`, and `find-grid` [unconditionally calls](https://github.com/at-import/Singularity/blob/c2ef7b6bcf7915ac460ab9021c3a743684ad4fe5/stylesheets/singularitygs/grids/_find.scss#L14) `find-object`.

Would send a PR, but i've long lost track of Singularity internals as they grew in complexity. Probably should just comment out the warning.
